pkg/internal/export/prom/prom.go

```diff
@@ -102,39 +102,60 @@ func newReporter(ctx context.Context, cfg *PrometheusConfig, ctxInfo *global.Con
 		cfg:         cfg,
 		promConnect: ctxInfo.Prometheus,
 		httpDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
-			Name:    HTTPServerDuration,
-			Help:    "duration of HTTP service calls from the server side, in seconds",
-			Buckets: cfg.Buckets.DurationHistogram,
+			Name:                            HTTPServerDuration,
+			Help:                            "duration of HTTP service calls from the server side, in seconds",
+			Buckets:                         cfg.Buckets.DurationHistogram,
+			NativeHistogramBucketFactor:     otel.DefaultHistogramBucketFactor,
```

Comment:
Nice! I know it's still early in this PR, but I wonder if we should make this optional? WDYT? Does it break things if a customer is using older Prometheus that doesn't support native histograms?

Reply:
Adding these parameters to histograms will keep compatibility with both formats (see [here](https://grafana.com/docs/mimir/latest/send/native-histograms/#migrate-from-classic-histograms). For OTEL i think is going to be another story.  

The reason that i opened the draft PR is to run integration tests. Can we assume if tests are passing means we didn't break histograms?

Reply:
Oh that's great! Yeah if tests pass it's all good!

Reply:
Seems the only issue is some formatting thing:
Error: pkg/internal/export/otel/common.go:59:1: ST1022: comment on exported var DefaultHistogramBucketFactor should be of the form "DefaultHistogramBucketFactor ..." (stylecheck)
// Default values for the histogram configuration